app/forms/journeys/additional_payments_for_teaching/nqt_in_academic_year_after_itt_form.rb

```diff
@@ -0,0 +1,69 @@
+module Journeys
+  module AdditionalPaymentsForTeaching
+    class NqtInAcademicYearAfterIttForm < Form
+      attribute :nqt_in_academic_year_after_itt, :boolean
+
+      validates :nqt_in_academic_year_after_itt,
+        inclusion: {
+          in: [true, false],
+          message: "Select yes if you are currently teaching as a qualified teacher"
+        }
+
+      def initialize(journey:, claim:, params:)
+        super
+
+        self.nqt_in_academic_year_after_itt = permitted_params.fetch(
+          :nqt_in_academic_year_after_itt,
+          claim.eligibility.nqt_in_academic_year_after_itt
+        )
+      end
+
+      def save
+        return false unless valid?
+
+        update!(
+          {
+            eligibility_attributes: {
+              nqt_in_academic_year_after_itt: nqt_in_academic_year_after_itt,
+              induction_completed: determine_induction_answer_from_dqt_record
+            }
+          }
+        )
+      end
+
+      def backlink_path
+        return unless page_sequence.in_sequence?("correct-school")
+
+        Rails
+          .application
+          .routes
+          .url_helpers
+          .claim_path(params[:journey], "correct-school")
+      end
+
+      private
+
+      def permitted_params
+        @permitted_params ||= params.fetch(:claim, {}).permit(
+          :nqt_in_academic_year_after_itt
+        )
+      end
```

Comment:
For another PR: this method can be removed as it's now on the base form and, by default, permits all the attributes defined on the form